Framework/DataHandling/src/Load.cpp

```diff
@@ -38,12 +38,7 @@ namespace {
  * @returns true if there is exactly one string, else false.
  */
 bool isSingleFile(const std::vector<std::vector<std::string>> &fileNames) {
-  if (fileNames.size() == 1) {
-    auto first = fileNames.cbegin();
-    if (first->size() == 1)
-      return true;
-  }
-  return false;
+  return fileNames.size() == 1 ? fileNames[0u].size() == 1 : false;
```

Comment:
Would `return fileNames.size() == 1 && fileNames[0].size() == 1` be slightly easier to read?

Reply:
Ah, that would be much better!